partcad/src/partcad/converter.py

```
@@ -0,0 +1,169 @@
+from OCP.StlAPI import StlAPI_Reader, StlAPI_Writer
+from OCP.BRepTools import BRepTools
+from OCP.BRepMesh import BRepMesh_IncrementalMesh
+from lib3mf import Wrapper, Position, Triangle
+from . import logging
+
+
+class CADConverter:
+    """
+    Handles CAD file conversion between supported formats (STEP, STL, 3MF, BREP).
+    """
+    CONVERTERS = {"step", "stl", "3mf", "brep"}
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)